Makefile

```diff
@@ -4,6 +4,7 @@ include vars.mk
 
 .PHONY: check
 check:
+	cd services/controller; pipenv install --dev; pipenv run yapf -rd ../..
```

Comment:
Make this part of the *cli* and *services/controller* Makefiles, and have them depend on the *environment* target.

Reply:
It'd be good to add a *format* target to *python.mk*, and have a root one that delegates to both.